drodrpg/DROD/GameScreen.cpp

```diff
@@ -2447,7 +2447,7 @@ void CGameScreen::OnClick(
 					{
 						ExploredRoom *pExpRoom = this->pCurrentGame->getExploredRoom(roomID);
 						if ((pExpRoom && pExpRoom->HasDetail()) ||
-							this->pCurrentGame->GetStoredMapStateForRoom(roomID) >= MapState::Preview)
+							CDbSavedGames::IsMoreDetailedMapState(this->pCurrentGame->GetStoredMapStateForRoom(roomID), MapState::NoDetail))
```

Comment:
Just checking -- should the second argument still be "Preview"?
(also below)

Reply:
These are correct, the map state has to be better than NoDetail in order to remotely view it. The new method does a greater than, the previous code here was greater than or equal to.